test/integration/targets/run_modules/runme.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# test running module directly
+library/test.py args.json
```

Comment:
```suggestion
python.py library/test.py args.json
```

Reply:
@mattclay trying to reproduce the original issue, which was 'direct module execution', I can add that as additional test, but would like the existing to stay

Reply:
The proposed change is still direct module execution. It does not go through Ansible. It does make sure the correct python interpreter is used though.